CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.14.1](https://github.com/brave/brave-ios/releases/tag/v1.14.1)
+
+ - Added images to new tab page. ([#1782](https://github.com/brave/brave-ios/issues/1782))
+ - Added wallet detail message for empty wallet. ([#1852](https://github.com/brave/brave-ios/issues/1852))
```

Comment:
nit: I think this was a regression right? I don't think we need to add `Added` and can used `Fixed`. Example:

```
 - Fixed wallet details not being displayed for empty wallets.
```